### ISSUE TITLE
How can I change the model input part by using the .csv file instead of the existing method?

### ISSUE THREAD

**Issue by qwer9007** [zhixuhao/unet#119]
Above all, thanks for your sharing.
Thanks to your kindness, I started to study U-Net.

I'm a beginner for U-Nets and Python. So my question might be look like foolish.
I have a question that How can I change the model input part by using the .csv file instead of the existing method?

I want to change data.py that you uploaded. But I couldn't change it well.

I added some codes into your data.py like this;

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import skimage.transform as trans
#ImageFile.LOAD_TRUNCATED_IMAGES = True

train = pd.read_csv('../data/datasets.csv', header=None, names=['image_train', 'image_label', 'value_label'], parse_dates=['image_train'], index_col=['image_train'])

x = DataFrame(train.drop('image_train',axis=1))
y = DataFrame(train['image_train']

x_train,y_train = train_test_split(x,y)

def adjustData(img,mask,flag_multi_class,num_class):
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

My csv file is structured as follows.
![image](https://user-images.githubusercontent.com/20736298/58011107-8e701b80-7b2c-11e9-9e1b-1d8ab99afadd.png)

Please give your idea. your tiny comments also help me a lot.

**Comment by deaspo** [comment]
Please check the following example --> [example ](https://www.pyimagesearch.com/2018/12/24/how-to-use-keras-fit-and-fit_generator-a-hands-on-tutorial/), may give you some insights
